skodaconnect/connection.py

```diff
@@ -1077,6 +1077,37 @@ async def getVehicleStatus(self, vin, smartlink=False):
             _LOGGER.warning(f'Could not fetch SmartLink vehicle status, error: {error}')
         return False
 
+    async def getAllStatistics(self, vin):
+        """Get all trips statistics."""
+        try:
+            await self.set_token('vwg')
+            short = await self.get(
+                urljoin(
+                    self._session_auth_ref_url[vin],
+                    f'fs-car/bs/tripstatistics/v1/{BRAND}/{COUNTRY}/vehicles/{vin}/tripdata/shortTerm?type=list&from=1970-01-01T00:00:00Z&to=2099-10-30T23:59:59Z'
+                )
+            )
+            cyclic = await self.get(
+                urljoin(
+                    self._session_auth_ref_url[vin],
+                    f'fs-car/bs/tripstatistics/v1/{BRAND}/{COUNTRY}/vehicles/{vin}/tripdata/cyclic?type=list&from=1970-01-01T00:00:00Z&to=2099-10-30T23:59:59Z'
```

Comment:
This is main difference between original example and API usage implemented in original connection.py